connexion/operations/abstract.py

```diff
@@ -424,9 +424,14 @@ def __validation_decorators(self):
                                      self.api,
                                      strict_validation=self.strict_validation)
         if self.body_schema:
+            try:
```

Comment:
Since this is OpenAPI 3 specific, we should probably implement this in `OpenAPIOperation` instead of `AbstractOperation`.